pkg/kubelet/status/status_manager.go

```diff
@@ -1023,23 +1023,26 @@ func normalizeStatus(pod *v1.Pod, status *v1.PodStatus) *v1.PodStatus {
 		normalizeTimeStamp(&condition.LastTransitionTime)
 	}
 
-	// update container statuses
-	for i := range status.ContainerStatuses {
-		cstatus := &status.ContainerStatuses[i]
-		normalizeContainerState(&cstatus.State)
-		normalizeContainerState(&cstatus.LastTerminationState)
+	normalizeContainerStatuses := func(containerStatuses []v1.ContainerStatus) {
+		for i := range containerStatuses {
+			cstatus := &containerStatuses[i]
+			normalizeContainerState(&cstatus.State)
+			normalizeContainerState(&cstatus.LastTerminationState)
+		}
+		// Sort the container statuses, so that the order won't affect the result of comparison
+		sort.Sort(kubetypes.SortedContainerStatuses(containerStatuses))
 	}
-	// Sort the container statuses, so that the order won't affect the result of comparison
-	sort.Sort(kubetypes.SortedContainerStatuses(status.ContainerStatuses))
 
-	// update init container statuses
-	for i := range status.InitContainerStatuses {
-		cstatus := &status.InitContainerStatuses[i]
-		normalizeContainerState(&cstatus.State)
-		normalizeContainerState(&cstatus.LastTerminationState)
-	}
-	// Sort the container statuses, so that the order won't affect the result of comparison
+	// Normalize and sort container statuses
```

Comment:
these comments (for all the three invocations) look pretty redundant, but please do NOT resubmit just to fix this

Reply:
+1, remove redundant comments